docker.test.js

```diff
@@ -111,6 +111,20 @@ test("parseImageName tests", () => {
     group: "shiftleft",
     name: "scan-java",
   });
+  expect(
```

Comment:
Sorry I meant docker tests workflow to generate sbom for containers

Reply:
Yes added